pootle/apps/pootle_store/models.py

```diff
@@ -872,10 +872,7 @@ def toggle_qualitycheck(self, check_id, false_positive, user):
                          unit=self, store=self.store, type=sub_type,
                          quality_check=check)
         sub.save()
-
-        # update timestamp
-        # log user action
-        self.save()
+        self.save(revision=Revision.incr())
```

Comment:
@ta2-1 - looking at it tho - i think i just wanted to be sure that the revision was incremented - maybe we dont need to - maybe - its just stats that need a kick